4_gap_analysis/src/Con_byState.R

```diff
@@ -0,0 +1,162 @@
+#   C (almost there, need more frequency) - sites with 6 samples per year, dissolved P, TP, NO3
+# (as you'll see, I've had to back the number of constituents down to 2 for A
+# and C to get any sites for AL)
+
+library(tidyverse)
+library(feather) # install with install.packages('feather')
+library(scipiper) # install with devtools::install_github('USGS-R/scipiper')
+
+state<-"Florida"
+
+# pull the files off of Google Drive - browser window for authentication should
+# pop up the first time you do this
+indicator_files <- dir('1_wqpdata/out/data', pattern='*\\.feather.ind$', full.names=TRUE) %>%
+  grep('(nitrate|phosphorus)', ., value=TRUE)
+sapply(indicator_files, scipiper::gd_get, USE.NAMES=FALSE)
+
+# read in the data and combine into one big file
+#data_files <- sapply(indicator_files, scipiper::as_data_file, USE.NAMES=FALSE)
+data_files <- dir('1_wqpdata/out/data', pattern=paste(state,'*', sep=''),full.names = T) %>%
+  grep('(nitrate|phosphorus)', ., value=TRUE)
+
+# combined data may not work
+combined_data <- bind_rows(
+  lapply(data_files, function(dfile) {
+    cname <- gsub(paste('1_wqpdata/out/data/',state,'_',sep=""), '', gsub("_00.\\.feather", '', dfile))
+    dat <- feather::read_feather(dfile) %>%
+     # mutate(ConstituentGroup=cname) %>%
+    # Below here are data conversion problems
+      mutate_at(vars(starts_with("Result")),funs(as.character)) %>% # FL
+      mutate_at(vars(ends_with("HeightMeasure.MeasureValue")),funs(as.character))# AL
+  })
+)%>%
+  mutate(ResultSampleFractionText=ifelse(is.na(ResultSampleFractionText),"Unknown",ResultSampleFractionText))
+
+# If we are using the ActivityStartDateTime field, get rid of NAs
+# Don't understand the replacement error warning error, lens of both
+# objects are 1
+combined_data$ActivityStartDateTime[is.na(combined_data$ActivityStartDateTime)]<-
+  as.POSIXlt(paste(combined_data$ActivityStartDate[is.na(combined_data$ActivityStartDateTime)],
+                   "12:00:00",sep=" "),format="%Y-%m-%d %H:%M:%S")
+
+# Crosswalk implementation, check with Gretchen
+# Does not include Organic P (236) and Polyphosphate (55)
+# List of Agency Names, Constitutents, and Filter Type from WQ Pull
+combined_data$OrganizationIdentifier[grepl("USGS*", combined_data$OrganizationIdentifier)]<-"USGS"
+# Subset conList to only P
+conList<-filter(combined_data,grepl("*(P|p)hos*",CharacteristicName)) %>% # Check for Orthophosphate and SRP with this
+  distinct(OrganizationIdentifier,CharacteristicName,ResultSampleFractionText) %>% #6/28 removed organization Name OrganizationFormalName,
+mutate(ResultSampleFractionText=ifelse(is.na(ResultSampleFractionText),"Unknown",ResultSampleFractionText))
+
+# Crosswalk based on Harmonization table
+xWalk1<-read.csv("andy/NH2012.csv",stringsAsFactors = F)
+# Crosswalk based on Agency List and acronyms
+xWalk2<-read.csv("andy/Crosswalk.csv",stringsAsFactors = F)
```

Comment:
remember to switch these two file paths to their new versioned locations, `4_gap_analysis/in/xxx.csv`